src/panels/lovelace/editor/card-editor/hui-dialog-edit-card.ts

```diff
@@ -44,6 +44,8 @@ export class HuiDialogEditCard extends LitElement {
   @property() private _saving: boolean = false;
   @property() private _error?: string;
 
+  @property() private _GUImode: boolean = true;
```

Comment:
What happens if a card doesn't have a GUI?

Reply:
Good point, how can I check that?

Reply:
Edit a stack

Reply:
Sorry what do you mean?

Reply:
No, but it seems to give a warning and we don't show the toggle button when there is a warning right?

Reply:
Yeah you are right, but how can I check for this warning if the custom element that detects it is not rendered yet?

Reply:
`hui-card-editor` should always be in control about the GUI state, not the dialog